frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -48,11 +48,11 @@ page-section.c-section(:title='L("Invite links")')
           )
             i.icon-ellipsis-v
         td.c-state
-          i18n.c-state-description {{ item.state.description }}
+          i18n.c-state-description {{ item.status.description }}
           i18n.c-state-expire(
-            v-if='item.state.expireInfo'
-            :class='item.state.isExpired ? "expired" : ""'
-          ) {{ item.state.expireInfo }}
+            v-if='item.status.expiryInfo'
+            :class='{ "expired": item.status.isExpired }'
```

Comment:
CSS classes that reflect a state should start with is `isExpired`

Reply:
fixed!